aiida/restapi/translator/nodes/node.py

```diff
@@ -346,12 +347,15 @@ def _get_subclasses(self, parent=None, parent_class=None, recursive=True):
             full_path_base = os.path.join(package_path, name)
 
             if is_pkg:
-                app_module = imp.load_package(full_path_base, full_path_base)
+                spec = importlib.util.spec_from_file_location(name, full_path_base)
+                app_module = importlib.util.module_from_spec(spec)
+                spec.loader.exec_module(app_module)
             else:
                 full_path = f'{full_path_base}.py'
-                # I could use load_module but it takes lots of arguments,
-                # then I use load_source
-                app_module = imp.load_source(f'rst{name}', full_path)
+                spec = importlib.util.spec_from_file_location(f'rst{name}', full_path)
+                app_module = importlib.util.module_from_spec(spec)
+                sys.modules[f'rst{name}'] = app_module
```

Comment:
same here (or perhaps I don't understand what this does).

is this some weird autocomplete of "firstname" in the editor?

Reply:
Yes, and I have changed them accordingly